mesonbuild/interpreter.py

```diff
@@ -1332,11 +1342,25 @@ def get_cross_property_method(self, args, kwargs):
                     'shared_library': shlib_kwargs,
                     'shared_module': shmod_kwargs,
                     'static_library': stlib_kwargs,
+                    'subdir': {'if_found'},
                     'subproject': {'version', 'default_options'},
                     'test': {'args', 'env', 'is_parallel', 'should_fail', 'timeout', 'workdir', 'suite'},
                     'vcs_tag': {'input', 'output', 'fallback', 'command', 'replace_string'},
                     }
 
+def is_disabled(args, kwargs):
+    for i in args:
+        if isinstance(i, DisablerHolder):
+            return True
+    for i in kwargs.values():
+        if isinstance(i, DisablerHolder):
+            return True
+        if isinstance(i, list):
+            for j in i:
+                if isinstance(j, DisablerHolder):
+                    return True
```

Comment:
This should just use `mesonlib.flatten()`, which is getting merged into `mesonlib.listify()` in https://github.com/mesonbuild/meson/pull/2375.